### DISCUSSION TITLE
use GeoJsonLayer for arcgis in CGCS2000 ( EPSG:4490 ) not correct zoom level

### DISCUSSION THREAD

**Discussion by leslie846933** [visgl/deck.gl#6817] [Q&A]
<img width="639" alt="QQ20220413-000056@2x" src="https://user-images.githubusercontent.com/21257579/163005156-98666298-9b70-4319-8857-acdf4389f6f6.png">
 
GeoJsonLayer data is same GeoJSON data , bule geometry is correct, but red geometry ( for deck.gl ) is not correct.
I try [https://github.com/visgl/deck.gl/discussions/5539](url) , [https://gist.github.com/tech-meppem/5a40e00b04220e924177791b4982be0f](url) , but it is not effect， what should i do ? thanks

**Comment by Pessimistress** [comment]
deck.gl only supports lng/lat and Cartesian coordinates. You need to convert your data to spherical coordinates if you wish to use another coordinate system.
